test_rmw_implementation/test/test_subscription.cpp

```diff
@@ -377,3 +377,92 @@ TEST_F(CLASSNAME(TestSubscriptionUse, RMW_IMPLEMENTATION), count_mismatched_subs
   EXPECT_EQ(RMW_RET_OK, ret) << rmw_get_error_string().str;
   EXPECT_EQ(0u, publisher_count);
 }
+
+TEST_F(CLASSNAME(TestSubscriptionUse, RMW_IMPLEMENTATION), take_sequence) {
+  rmw_ret_t ret;
```

Comment:
nit: Move this variable declaration to line 387 where it is initialized

Reply:
Updated on dc8ab77